services/validateToken.js

```diff
@@ -0,0 +1,16 @@
+const jwt = require('jsonwebtoken');
+
+const secretKey = process.env.JWT_SECRET;
+
+function validateToken(token) {
+  try {
+    jwt.verify(token, secretKey);
```

Comment:
we can reuse function from utils/helpers that we use in past. And also I this this function will return true and false so we don't need to create condition we just return what ever this function return. But test this